airflow/api_connexion/openapi/v1.yaml

```diff
@@ -3097,7 +3099,23 @@ components:
         status:
           $ref: '#/components/schemas/HealthStatus'
         latest_scheduler_heartbeat:
-          description: The time the scheduler last do a heartbeat.
+          description: The time the scheduler last does a heartbeat.
```

Comment:
This should be past tense?

Reply:
In that case, I think "The time the scheduler last did a heartbeat." could suit better?

Reply:
How about "The time the scheduler did the last heartbeat"

Reply:
Both make sense (slightly different meaning but both correct)

Reply:
I went with the first one, updated for both scheduler and triggerer description :)